test/FsAutoComplete.Tests.Lsp/CodeLensTests.fs

```diff
@@ -11,6 +11,7 @@ open Utils.Utils
 open Utils.CursorbasedTests
 open Utils.Tests.TextEdit
 open Newtonsoft.Json.Linq
+open Helpers.Expecto.ShadowedTimeouts
```

Comment:
We have to add this to all test files if we want the shadowed versions of `testCase` and friends that have the timeout embedded in them. Otherwise I could rename all the testCases to use the timeout test explicitly